backend/resolver/aws/aws.go

```diff
@@ -168,10 +168,15 @@ func (r *res) Search(ctx context.Context, typeURL, query string, limit uint32) (
 		if err != nil {
 			return nil, err
 		}
+
 		if ok {
-			return r.instanceResults(ctx, patternValues["account"], patternValues["region"], []string{patternValues["instance_id"]}, limit)
+			// in the case of account not found, we continue to the below flow of only being concerned with the ID, rather than the account and region
+			if _, ok := r.client.AccountsAndRegions()[(patternValues["account"])]; ok {
```

Comment:
Note that I chose to use the `AccountsAndRegions` function since `Accounts` gives me a list rather than a map and I wanted a map. I thought about this and realized that there are probably very few accounts, so it wouldn't really matter if it was a list or map. Or I could add a new function similar to `Accounts` but it returns a map instead. Many options here.